comparisons/components/image_slider/a11y.txt

```diff
@@ -0,0 +1 @@
+Usage: Screen reader accessible, no user control over images.
```

Comment:
I might not use the phrase screen reader accessible. You might say this is presentational and skipped by the screen reader. Using an empty string in your alt tag, `alt=""`, explicitly tells the screen reader to ignore the image. Kind of like saying "Hey this is just decoration, it's not super important for the user experience".